docs/data/charts/sparkline/CustomDomainYAxis.tsx

```diff
@@ -0,0 +1,90 @@
+import * as React from 'react';
+import TextField from '@mui/material/TextField';
+import MenuItem from '@mui/material/MenuItem';
+import Stack from '@mui/material/Stack';
+import Box from '@mui/material/Box';
+import Typography from '@mui/material/Typography';
+import { Theme } from '@mui/material/styles';
+import { SparkLineChart } from '@mui/x-charts/SparkLineChart';
```

Comment:
To avoid the blank line in the JavaScript generated demo?

```suggestion
import { SparkLineChart } from '@mui/x-charts/SparkLineChart';
import { Theme } from '@mui/material/styles';
```